crates/aptos-rest-client/src/lib.rs

```diff
@@ -566,7 +566,8 @@ impl Client {
         F: Fn(HashValue) -> Fut,
         Fut: Future<Output = AptosResult<WaitForTransactionResult<T>>>,
     {
-        const DEFAULT_DELAY: Duration = Duration::from_millis(500);
+        // note: make this configurable
+        const DEFAULT_DELAY: Duration = Duration::from_millis(100);
```

Comment:
I feel by doing this we make the risk of production use cases of this client hit rate limits much higher, could we maybe revert this unless we're confident we won't run into such problems.

Reply:
Yeah, I would agree that we probably want to make this configurable via input rather than continue being defaulted like this, but also it's some of the things that need to be cleaned up in the rust SDK